scripts/mbedtls_dev/ecp_common.py

```diff
@@ -0,0 +1,22 @@
+"""Common features for ecp in test generation framework."""
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from . import bignum_common
+
+class EcpOperationCommon(bignum_common.ModOperationCommon):
```

Comment:
I currently see no reason for a dedicated ecp_common.py file. Are we expecting a lot of ecp specific utilities functions? 

It would be better to not increase complexity of the scripts and make ecp tests inherit from `bignum_common.ModOperationCommon`.

Reply:
There will be a lot of ecp specific functions, but the ECC modules have much less common than the Bignum modules and we probably won't need an ecp_common.py. I agree that using `bignum_common.ModOperationCommon` would be more simple.

The `ecp_curves` module is halfway between Bignum and ECP and it is acceptable if the boundaries are somewhat blurred.